tutor/templates/build/openedx/Dockerfile

```diff
@@ -124,14 +142,17 @@ RUN useradd --home-dir /openedx --create-home --shell /bin/bash --uid ${APP_USER
 USER ${APP_USER_ID}
 
 # https://hub.docker.com/r/powerman/dockerize/tags
-COPY --from=docker.io/powerman/dockerize:0.19.0 /usr/local/bin/dockerize /usr/local/bin/dockerize
-COPY --chown=app:app --from=code /openedx/edx-platform /openedx/edx-platform
+COPY {% if is_buildkit_enabled() %}--link {% endif %}--from=docker.io/powerman/dockerize:0.19.0 /usr/local/bin/dockerize /usr/local/bin/dockerize
+COPY --chown=app:app --from=edx-platform / /openedx/edx-platform
```

Comment:
`COPY --link` caused docker to crash on my machine  when building[^1], and it was resolved by adding following the line in Dockerfile
`#syntax=docker/dockerfile:1.4` ref[^2]. Any yes BuildKit is enabled by default for me. 

[^1]: Darwin M2 docker  engine: 20.10.24
[^2]: https://stackoverflow.com/a/75199570/5532723

Reply:
Thanks a lot Ghassan, this is super useful.